plugins/woocommerce-admin/client/marketplace/components/product-card/product-card.scss

```diff
@@ -269,6 +284,30 @@
 				line-height: 1.5;
 			}
 		}
+
+		&.is-sponsored::before {
+			content: ' ';
+			height: 46px;
+			width: 46px;
+			background: #09B58540;
```

Comment:
Maybe this should be added to `../../stylesheets/_variables.scss`?

Reply:
Done in 920444cecd9d0f7d288caff374fa7a443b3da50a. Thanks!